src/client/trackers/real-user-monitoring-for-performance.js

```diff
@@ -7,7 +7,7 @@ export const realUserMonitoringForPerformance = () => {
 	if (!userIsInCohort(cohortPercent)) return;
 
 	// For browser compatibility @see: https://mdn.github.io/dom-examples/performance-apis/perf-api-support.html
-	if (!'PerformanceLongTaskTiming' in window || !'ttiPolyfill' in window) return;
+	if (!'PerformanceLongTaskTiming' in window || !'perfMetrics' in window) return;
```

Comment:
Note: `perfMetrics` is added to `window` scope  by the `first-input-delay` code. 

@see: https://github.com/GoogleChromeLabs/first-input-delay/blob/master/src/first-input-delay.js#L170